functions/go/set-namespace/README.md

```diff
@@ -4,76 +4,47 @@
 
 <!--mdtogo:Short-->
 
-The `set-namespace` function update or add namespace to all namespaced
-resources. Kubernetes supports multiple virtual clusters backed by the same
-physical cluster through namespaces.
-
-Namespaces are often used in the following scenarios:
```

Comment:
We should follow the template: https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/master/functions/go/_template/README.md#overview

Some of these lines can be kept. If you don't think they should be part of the short description (i.e. between `<!--mdtogo:Short-->` and `<!--mdtogo-->`), we can keep them between `<!--mdtogo-->` and `<!--mdtogo:Long-->`

Reply:
moved between <!--mdtogo--> and <!--mdtogo:Long-->